app/javascript/legacy/member-facing/backbone/petition_and_scroll_to_consent.js

```diff
@@ -24,6 +25,8 @@ const PetitionAndScrollToConsent = Backbone.View.extend({
   //    after a successfull submission of the action form instead of
   //    the doing the redirect.
   initialize(options = {}) {
+    this.user_id =
+      window.champaign.personalization.member.id || Cookie.get('_facebook_uid');
```

Comment:
Where is `_facebook_uid` set?

Reply:
It should be __bpmx cookie. I will update it